queries/hoon/highlights.scm

```diff
@@ -0,0 +1,32 @@
+(number) @number
+
+(string) @string
+
+[
+  "("
+  ")"
+  "["
+  "]"
+]  @punctuation.bracket
+
+[
+  (coreTerminator)
+  (seriesTerminator)
+] @punctuation.delimiter
+
+
+(rune) @keyword
```

Comment:
from the looks of it, isn't this an operator (or punctuation.(delimiter|special)?

Reply:
correct, a rune should be an operator